packages/ui/src/molecules/PaymentMethods/PaymentMethods.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import Label from '../../atoms/Label'
+
+export interface PaymentMethodsProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Title of the payment methods section (e.g.: "Payment methods",
+   * "Accepted Cards").
+   */
+  title?: ReactNode
+  /**
+   * Children will receive the flags to be displayed in the payment
+   * methods section (e.g.:, visa, mastercard, etc).
+   */
+  children: ReactNode
+}
+
+const PaymentMethods = ({
+  testId = 'store-payment-methods',
+  title,
+  children,
+}: PaymentMethodsProps & { children: ReactNode }) => {
```

Comment:
I wonder if the ` & { children: ReactNode }` bit is needed considering `PaymentMethodsProps` already specified the `children` property!

```suggestion
}: PaymentMethodsProps) => {
```

Reply:
Good catch, I changed the `children` and forgot to take that away.